components/upload/Upload.tsx

```diff
@@ -40,11 +35,7 @@ export default class Upload extends React.Component<UploadProps, any> {
     supportServerRender: true,
   };
 
-  static contextTypes = {
-    antLocale: React.PropTypes.object,
-  };
-
-  context: UploadContext;
+  static contextTypes = UploadLocalizable.contextTypes;
```

Comment:
To solve issues which similar to: https://github.com/ant-design/ant-design/issues/5484